### PR TITLE
feat(ui5-popup): add accessibleNameRef property

### PR DESCRIPTION
**Pull request by dimovpetar**
Affects Dialog, Popover and ResponsivePopover.

### CODE REVIEW

File: packages/main/src/Popup.js [hunk 1]
```diff
@@ -80,6 +81,19 @@ const metadata = {
 			defaultValue: undefined,
 		},
 
+		/**
+		 * Defines the IDs of the elements that label the component.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.0.3
```
Comment (fifoosid) [inline packages/main/src/Popup.js:90]: since 1.0.0

File: packages/main/src/Popup.js [hunk 2]
```diff
@@ -535,7 +549,7 @@ class Popup extends UI5Element {
 	 * @protected
 	 */
 	get _ariaLabel() {
-		return this.accessibleName || undefined;
+		return getEffectiveAriaLabelText(this) || undefined;
```
Comment (fifoosid) [inline packages/main/src/Popup.js:552]: When using ```getEffectiveAriaLabelText``` you don't need the fallback value of undefined. This case is handled.

**Review by fifoosid — CHANGES_REQUESTED** [review]
If possible, it might be a good idea to add a test for the responsive popover as well

File: packages/main/src/Popup.js (hunk 2 above)
Comment (dimovpetar) [reply packages/main/src/Popup.js]: Done

File: packages/main/src/Popup.js (hunk 1 above)
Comment (dimovpetar) [reply packages/main/src/Popup.js]: I assume 1.1.0, like in the RadioButton. Done

**Comment by dimovpetar** [comment]
> If possible, it might be a good idea to add a test for the responsive popover as well

Couldn't find a way to mock phone. The desktop part is already covered

**Review by fifoosid — APPROVED** [review]

**Review by GerganaKremenska — APPROVED** [review]

**Review by fifoosid — APPROVED** [review]

**Pull request merged**